apps/asap-server/src/utils/sync-google-event.ts

```diff
@@ -36,12 +36,12 @@ export const syncEventFactory = (
     const { error, value } = schema.validate(eventPayload);
 
     if (error) {
-      logger('Ignored event update, validation error:', error);
+      logger.error('Ignored event update, validation error:', error);
       return Promise.reject(error);
     }
 
     const googleEvent = value as GoogleEvent;
-    logger('google event', googleEvent);
+    logger.info('google event', googleEvent);
```

Comment:
Not sure if this `info`s shouldn't be `debug`. They make more sense for debugging purposes. What do you think?